### ISSUE TITLE
[TypeScript] `makeCapabilityInstance` wrongly defined on 3.0.5

### ISSUE THREAD

**Issue by OlivierZal** [athombv/web-api-issues#33]
4 occurrences in `homey-api.d.ts`:

```
makeCapabilityInstance(
    capabilityId: string,
    listener: {
        value: number | boolean | string,
    },
): HomeyAPIV3.ManagerDevices.Device.DeviceCapability;
```

should rather be:

```
makeCapabilityInstance(
    capabilityId: string,
    listener: (
        value: number | boolean | string,
    ) => Promise<void>,
): HomeyAPIV3.ManagerDevices.Device.DeviceCapability;
```

**Comment by jeroenwienk** [comment]
Fixed in 3.0.7

**Comment by OlivierZal** [comment]
Confirmed
